specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,39 @@
+# Commitment Discount Type
+
+A commitment-based discount is a contractual commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options and time frames.
+
+Commitment Discount Type indicates whether the commitment-based discount identified in the CommitmentDiscountId column is based on usage quantity or cost (aka "spend").
+
+The CommitmentDiscountType column MUST be present in the billing data. This column MUST be of type String, MUST be null when CommitmentDiscountId is null, and MUST NOT be null when CommitmentDiscountId is not null. The CommitmentDiscountType MUST be one of the allowed values.
+
+## Column ID
+
+CommitmentDiscountType
+
+## Display name
+
+Commitment Discount Type
+
+## Description
+
+Indicates whether the commitment-based discount identified in the CommitmentDiscountId column is based on usage quantity or cost (aka "spend").
+
+## Content constraints
+
+|    Constraint   |      Value       |
+|:----------------|:-----------------|
+| Column required | True             |
+| Data type       | String           |
+| Allows nulls    | True             |
+| Value format    | list-of-values   |
+
+Allowed values:
+
+| Value      | Description                                                                                                                                                                   |
+|:---------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spend   | Commitment-based discounts that require a predetermined amount of spend.    |
+| Usage   | Commitment-based discounts that require a predetermined amount of usage.     |
```

Comment:
```suggestion
|:--------|:--------------------------------------------------------------------------------|
| Spend   | Commitment-based discounts that require a predetermined amount of spend.    |
| Usage   | Commitment-based discounts that require a predetermined amount of usage.    |
```

Reply:
Personally, I think Spend-commitment and Usage commitment are more descriptive terms than just Spend and Usage. Curious what others think. I think that may have gotten updated recently so I assume it was discussed.

Reply:
Personally, I find "shorter is better" for column values, as the column name gives the context for the values.
Another example is the PricingModel PR: I would find cumbersome if every value of the Pricing Model column is suffixed with "Pricing Model".
For column names, I find that values being more explicit and descriptive is indeed better.

Reply:
I would vote for Usage and Spend as column name is descriptive